Makefile

```diff
@@ -37,6 +37,9 @@ STRESSFLAGS  :=
 DUPLFLAGS    := -t 100
 
 ifeq ($(STATIC),1)
+# The Go resolver should work well enough for now; better than musl, anyway.
```

Comment:
It's not clear that gliderlabs/docker-alpine#8 is the same issue. That's about some options in `resolv.conf` which is an acknowledged missing feature in musl. Here the problem is not about DNS search paths, it's simple entries in /etc/hosts (which I think are supposed to work, although musl is clearly designed for use with a local dns server instead of complex glibc-style resolution).